linode/manager
# upcoming: [DI-20800] - CSS for widget size control and feedbacks

## Description 📝
CSS updates according to UX feedback for widgets and main bar

## Changes  🔄

1. Fixed widget paper wrapper height
2. Fixed legend table height and overflow on scroll
3. Removed divider between filters and graph inside widget
4. Main Bar consistency according to UX margin (24px), divider color update
5. Placeholder update according to suggestion

## Target release date 🗓️
01-10-2024

## Preview 📷
| Before  | After   |
| ------- | ------- |
|![Screenshot 2024-09-17 at 4 53 46 PM](https://github.com/user-attachments/assets/b626f63e-daae-498d-843b-0fbc966ec3f6)|![Screenshot 2024-09-17 at 4 28 06 PM](https://github.com/user-attachments/assets/85a10550-d753-40f4-9dca-e5ccd822e720)|
|![Screenshot 2024-09-17 at 4 54 16 PM](https://github.com/user-attachments/assets/a81a79c3-d7f1-4b68-93a3-5e0c894ab419)|![Screenshot 2024-09-17 at 4 47 57 PM](https://github.com/user-attachments/assets/64e50676-8c6e-4e21-b1a2-5089586aedb3)|

## How to test 🧪

1. As some of endpoints are not available enable mock service
2. Go to monitors tab
3. Select dashboard and filters, you will be able to see widgets and CSS fixes

## As an Author I have considered 🤔

*Check all that apply*
